RecoTracker/Configuration/python/customiseForRunI.py

```diff
@@ -109,4 +109,14 @@ def customiseForRunI(process):
     process.SiStripClusterChargeCutTight.value = -1.
     process.SiStripClusterChargeCutLoose.value = -1.
 
+
+    if hasattr(process, 'TrackingDQMSourceTier0Common') and 'TrackSeedMonjetCoreRegionalStep' in process.TrackingDQMSourceTier0Common.moduleNames():
+        process.TrackingDQMSourceTier0Common.remove(process.TrackSeedMonjetCoreRegionalStep)
```

Comment:
there could have been a more elegant way that doesn't introduce "DQM" into pure Reco configs.